index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<htmllang="en">
```

Comment:
Looks like a space was accidentally deleted here - should be `<html lang="en">`.